Header header

string type
uint16 id
uint32 left
uint32 right
uint32 top
uint32 bottom
float32[] mask_1d
